source/includes/gcp/_routers.md

```diff
@@ -160,7 +160,7 @@ Create a new router
 | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | `name`<br/>_string_         | The display name of the router.                                                                                                 |
 | `shortRegion`<br/>_string_  | A short version of the region name.                                                                                             |
-| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                        |
+| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                                 |
 | `bgp.asn`<br/>_string_      | The BGP Autonomous System Number (ASN) Number that is to be assigned to this router. _Must be a valid ASN between 64512–65534_. |
```

Comment:
"Number (ASN) Number". Is it alright if we ditch the second time we use the word "Number"?

Reply:
Ofcourse xD. My bad;
Fixed in 7e73a1e91a08ace8091352d9798a376915815c5a